dali/base/dadfs.cpp

```diff
@@ -2782,6 +2790,14 @@ class CDistributedFileBase : implements INTERFACE, public CInterface
         root->removeProp("Attr");
         return NULL;
     }
+    IPropertyTree * resetHistory(IPropertyTree *history=NULL)
+    {
+        if (!history)
+            return queryAttributes().setPropTree("History",createPTree("History"));
+        else
+            queryAttributes().removeTree("History");
+        return nullptr;
+    }
```

Comment:
A few things odd/wrong with this resetHistory function.
passes in a param, which could be boolean, which either deletes history and replaces with empty node, or just deletes history.
Then returns either the new empty tree, or null.
resetTree takes a IPT but is being passed a string.

Shouldn't this just be:
```
void resetHistory()
{
    queryAttributes().removeProp("History");
}
```

?